app/controllers/comments_controller.rb

```diff
@@ -4,7 +4,7 @@ class CommentsController < ApplicationController
 
   def create
     @comment = current_user.comments.build comment_params
-    flash[:danger] = t "msg.comment_fail" unless @comment.save
+    @comment.save
```

Comment:
k xử lí thành công hay thất bại ah e

Reply:
Em fix lại toàn bộ xử lý cmt bằng ajax lại rồi ạ ( bỏ chung vào commit này ).  ( Mặc dù comment trước em đc merge rồi mà em thấy cmt k dùng ajax thì ko mượt  :D ). Em đã edit ạ, dùng ajax thì mới báo lỗi form trong comment đc.